src/ai/susi/server/api/cms/GetSkillsByAuthorEmail.java

```diff
@@ -0,0 +1,141 @@
+package ai.susi.server.api.cms;
+
+import ai.susi.DAO;
+import ai.susi.EmailHandler;
+import ai.susi.json.JsonObjectWithDefault;
+import ai.susi.mind.SusiSkill;
+import ai.susi.server.*;
+import ai.susi.tools.TimeoutMatcher;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.mail.Folder;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.util.*;
+import java.util.regex.Pattern;
+
+/**
+ * Created by @Akshat-Jain on 19/06/17.
+ * API Endpoint to get all skills by an author using his email.
+ * http://127.0.0.1:4000/cms/getSkillsByAuthorEmail.json?author_email=akjn11@gmail.com
+ */
+public class GetSkillsByAuthorEmail extends AbstractAPIHandler implements APIHandler {
```

Comment:
Please do not create a new API for each and every case. This one is very similar to GetSkillsByAuthor.java.
Please merge your code with GetSkillsByAuthor and extend that class. Distinguish the cases by using different post/get parameters to trigger the way how to retrieve the skill.